app/mailers/generic_mailer.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class GenericMailer < ApplicationMailer
+  default from: %("Mailgun Sandbox" \
+    <postmaster@sandbox8f696e611a6e4906a977c007f8971322.mailgun.org>)
+
+  def drop_duty(duty)
+    @duty = duty
+    mail(to: all_users_with_name,
+         subject: 'Duty notification: '\
+                  "#{duty.time_range.start_time.strftime('%H%M')}-"\
+                  "#{duty.time_range.end_time.strftime('%H%M')} on"\
+                  "#{duty.date.strftime('%a, %d %b %Y')} at"\
+                  "#{duty.place.name}")
+  end
+
+  def problem_report(problem)
+    @problem = problem
+    mail(to: cell_users_with_name('Technical'),
+         subject: 'New computer problem')
+  end
+
+  private
+
+  def users_with_name(users)
+    users&.pluck(:username, :email)&.map do |u|
+      %("#{u[0]}" <#{u[1]}>)
+    end
+  end
+
+  def all_users_with_name
+    users_with_name(User)
```

Comment:
lol use `User.all` here to make the intent clearer

Reply:
But would that load all users into memory?

Reply:
`User.pluck` is no different than `User.all.pluck` lol

Reply:
Anyways, another reason I recommended abstracting the email out is so that you can load the Users in batches (say, 200) then send email incrementally. Probably not much use here, but we never know if commIT recruits 1000 people. :)

Reply:
Hmm technically you don't really need to since there is `deliver_later` which, if I understand correctly would send emails asynchronously using activejobs or sth like that. But sure, abstracting the email out sounds better.